internal.h

```diff
@@ -48,6 +48,17 @@ extern bool verbose;
 #define NOTICE(fmt, args...) { if (debug) printf (fmt, ## args); else syslog (LOG_NOTICE, fmt, ## args); }
 #define ERROR(fmt, args...) { if (debug) printf (fmt, ## args); else syslog (LOG_CRIT, fmt, ## args); }
 
+typedef enum
+{
+    LOG_NONE                    = 0,
+    LOG_POST                    = (1 << 0),  /* Log POST requests */
+    LOG_PUT                     = (1 << 1),  /* Log PUT requests */
+    LOG_PATCH                   = (1 << 2),  /* Log PATCH requests */
+    LOG_DELETE                  = (1 << 3),  /* Log DELETE requests */
+    LOG_GET                     = (1 << 4),  /* Log GET requests */
+    LOG_HEAD                    = (1 << 5),  /* Log HEAD requests */
+} logging_flags;
+
```

Comment:
a global "int logging" would be simpler ... so then the code could just check "if (logging & LOG_POST)" etc

Reply:
Done